flair/models/lemmatizer_model.py

```diff
@@ -560,7 +562,8 @@ def predict(
                         hypothesis_scores = torch.cat(self.beam_size * [scores], dim=1) + log_probabilities
                         # print(hypothesis_scores)
 
-                        # reshape to vector of size (batch_size, beam_size*beam_size), each row contains beam_size*beam_size scores of the new possible hypothesis
+                        # reshape to vector of size (batch_size, beam_size*beam_size),
+                        # each row contains beam_size*beam_size scores of the new possible hypothesis
                         hypothesis_scores_per_token = hypothesis_scores.view(number_tokens, self.beam_size ** 2)
```

Comment:
```suggestion
                        hypothesis_scores_per_token = hypothesis_scores.view(number_tokens, self.beam_size**2)
```
@alanakbik  with the new release, of black, this line changes.
I see this is annoying when the build changes just because a dev library changes behaivour. I will think of a solution in the weekend.

Reply:
Ah wow thanks for spotting this. I guess everyone then needs to update to the newest version of black.